### PR TITLE
Fixed an unhandled exception in aker.py

### PR DESCRIPTION
**Pull request by pewo**
Small fix to handle a exception in aker.py

### CODE REVIEW

File: aker.py [hunk 1]
```diff
@@ -154,7 +154,17 @@ def init_connection(self, host):
...
+            logging.debug("Could not connect session"+ " (" + repr(e) + ")")
+            print(e)
+            raw_input("Press Enter to continue...")
+            session.stop_sniffer()
+            self.tui.restore()
+            self.tui.hostlist.search.clear()  # Clear selected hosts
+            return None
+
         try:
             session.start_session()
```
Comment (EoleDev) [inline aker.py]:
```suggestion
            session.connect(screen_size)
            session.start_session()
        except Exception as e:
            logging.error(
                "Core: Error during connection or starting session UUID {0} for user {1} : {2} ".format(
                    session_uuid, self.posix_user, e.message))
            print(e)
            raw_input("Press Enter to continue...")
```

**Review by EoleDev — CHANGES_REQUESTED** [review]
Hi @pewo,
I think, it's best to put the session connection and start in the same try. It would allow to catch and print the error in the two cases and avoid the code redundancy in the error handling.
I did a quick suggestion on your code and changed two things : 
  - the one mentioned above
  - the logging error message. I changed it to be consistent with the project way of loggging.

@anazmy I would be glad if you could give your opinion about the print and raw_input part. Is there a better way to do it, or is already the best ?

I am not quite proficient with the review system of github, so if I made an error, feel free to correct it!

File: aker.py (hunk 1 above)
Comment (pewo) [reply aker.py]: Looks much better!

**Review by EoleDev — APPROVED** [review]
All seems good for me.
We will need your review @anazmy !
And if you may merge it after, we would be glad!

**Comment by anazmy** [comment]
The PR has a problem if the failed/offline server is not the first one tried. i.e. try connecting to an online server first then to an offline server. there is still an error.

Need to look into that further.

**Comment by anazmy** [comment]
Fixed the stdin issue by modifying `SSHClient.py`. The PR looks good now.

Thx a lot @pewo and @EoleDev

**Comment by anazmy** [comment]
Fixes #98

**Pull request merged**